tests/scripts/deploy_admission_controller.sh

```diff
@@ -34,7 +35,7 @@ export SERVICE_NAME="rook-ceph-admission-controller"
 echo "$BASE_DIR"
 
 kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/$CERT_VERSION/cert-manager.yaml
-timeout 150 sh -c 'until [ $(kubectl -n cert-manager get pods --field-selector=status.phase=Running|grep -c ^cert-) -eq 3 ]; do sleep 1; done'
+timeout 180 sh -c 'until [ $(kubectl -n cert-manager get pods --field-selector=status.phase=Running|grep -c ^cert-) -eq 3 ]; do sleep 1; done'
```

Comment:
It would help to see some status with each iteration. Could you also add a print statement for the cert pod status as part of the loop? Also, maybe sleep 5 instead of sleep 1 to reduce the number of prints.